lib/src/track/audio_visualizer_web.dart

```diff
@@ -52,7 +52,9 @@ class AudioVisualizerWeb extends AudioVisualizer {
           frequencies = frequencies.sublist(
               options.loPass!.toInt(), options.hiPass!.toInt());
 
-          final normalizedFrequencies = normalizeFrequencies(frequencies);
+          final normalizedFrequencies = visualizerOptions.smoothTransition
+              ? normalizeFrequencies(frequencies)
+              : frequencies;
```

Comment:
Ah I don't think this is the right place to make the change for web. Instead, I think we need to lower the `smoothingTimeConstant` in `AudioAnalyserOptions` in `_audio_analyser.dart`